### PR TITLE
Added documentation for AUTO_UPDATE_CONFIG

### PR DESCRIPTION
**Pull request by jainishshah17**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Changelog.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Fix for https://github.com/jfrog/charts/issues/107

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
https://github.com/jfrog/charts/issues/107

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/artifactory-ha/ReverseProxyConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        proxy_set_header    X-Forwarded-Proto $http_x_forwarded_proto;
+        proxy_set_header    Host              $http_host;
+        proxy_set_header    X-Forwarded-For   $proxy_add_x_forwarded_for;
+        }
+    }
+    ```
+       
+*   Once Artifactory is deployed. Get Nginx pod name by running `kubectl get pods`.
+
+*   Copy `artifactory.conf` file to Nginx pod using following command:
```
[the remark this answers is not in the document]
Comment (jainishshah17) [reply stable/artifactory-ha/ReverseProxyConfiguration.md]: @eldada Let me change this to configMap

File: stable/artifactory-ha/ReverseProxyConfiguration.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    ```
+    Docker Access Method: SubDomain
+    Server Provider: Nginx
+    Internal Hostname: $ARTIFACTORY_SERVICE_NAME
+    Public Server Name: $DOMAIN_NAME
+    SSL Key Path: /var/opt/jfrog/nginx/ssl/tls.key (If SSL Cert is provided via Secret)
+    SSL Certificate Path: /var/opt/jfrog/nginx/ssl/tls.crt (If SSL Cert is provided via Secret)
+    ```
+*   Provide appropriate values and save configuration.
+*   Once configuration is saved Nginx will automatically fetch reverse proxy configuration snippet from Artifactory and apply it immediately.
```
[the remark this answers is not in the document]
Comment (jainishshah17) [reply stable/artifactory-ha/ReverseProxyConfiguration.md]: @eldada done

File: stable/artifactory-ha/ReverseProxyConfiguration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jainishshah17) [reply stable/artifactory-ha/ReverseProxyConfiguration.md]: @eldada @rimusz done

**Comment by eldada** [comment]
One last thought - if `nginx.customArtifactoryConfigMap` is set we should force `SKIP_AUTO_UPDATE_CONFIG=true`. 
If not, a user might forget to set `nginx.env.skipAutoConfigUpdate=true` and Nginx will override config, causing the custom config to be lost. User will never figure it out.
WDYT?

**Comment by jainishshah17** [comment]
@eldada I agree

**Pull request merged**